internal/run/source.go

```diff
@@ -128,7 +128,7 @@ func (w *Workflow) RunSource(ctx context.Context, parentStep *workflowTracking.W
 		} else if !isSingleRegistrySource(w.workflow.Sources[sourceID]) && w.workflow.Sources[sourceID].Registry == nil {
 			return "", nil, fmt.Errorf("invalid workflow lockfile: no registry location found for source %s", sourceID)
 		} else if w.workflow.Sources[sourceID].Registry != nil {
-			orgSlug, workspaceSlug, registryNamespace, err = w.workflow.Sources[sourceID].Registry.ParseRegistryLocation()
+			orgSlug, workspaceSlug, registryNamespace, _, err = w.workflow.Sources[sourceID].Registry.ParseRegistryLocation()
```

Comment:
Seems strange to add this but not use it anywhere?

Reply:
Well the problem is we were including tags in the namespace when we shouldn't.

We could just not return it. But seems like it's maybe useful to make it available